config/application.rb

```diff
@@ -17,5 +17,12 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
```

Comment:
What does `origins '*'` do? Intuitively, I'm guessing this opens things wide up, basically defeating the purpose of CORS. Should this not be a list of known origins? I could be wrong here, feel free to school me

Reply:
Correct. I left it that way because I don't currently have a defined port on localhost for my React server. Can use:

`origins 'localhost:3001', '0.0.0.0:3001'`